src/Organizations/OrganizationDetails/OrganizationDetails.js

```diff
@@ -47,7 +47,7 @@ const OrganizationDetails = ({
   const [expandAll, sections, toggleSection] = useAccordionToggle({
     [ORGANIZATION_SECTIONS.summarySection]: true,
     [ORGANIZATION_SECTIONS.contactInformationSection]: false,
-    [ORGANIZATION_SECTIONS.contactPeopleSection]: false,
+    [ORGANIZATION_SECTIONS.contactPeopleSection]: true,
```

Comment:
is it required?

Reply:
there is a test case section should be opened by default

Reply:
but if tests were not running for some time, it might be an obsolete test and business requirement has been changed since. Maybe we need to check if there are any bugs on open this section, but I suppose it's better to leave this closed

Reply:
it's opened on all envs so case is correct